pkg/controllers/management/k3supgrade/template.go

```diff
@@ -0,0 +1,163 @@
+package k3supgrade
+
+import (
+	"github.com/rancher/system-upgrade-controller/pkg/apis/upgrade.cattle.io"
+	planv1 "github.com/rancher/system-upgrade-controller/pkg/apis/upgrade.cattle.io/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubectl/pkg/describe"
+)
+
+const k3sMasterPlan = `---
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: k3s-master-plan 
+  namespace: system-upgrade
+spec:
+  concurrency: 1
+  version: v1.17.2-k3s1
+  nodeSelector:
+    matchExpressions:
+      - {key: k3s-upgrade, operator: Exists}
+  serviceAccountName: default
+  drain:
+    force: true
+  upgrade:
+    image: rancher/k3s-upgrade:latest`
+
+// https://github.com/rancher/k3s-upgrade/pull/9
+const k3sWorkerPlan = `---
+apiVersion: upgrade.cattle.io/v1
+kind: Plan
+metadata:
+  name: k3s-worker-plan
+  namespace: system-upgrade
+spec:
+  concurrency: 1
+  version: v1.17.2-k3s1
+  # The prepare init container is run before cordon/drain which is run before the upgrade container.
+  # Shares the same format as the "upgrade" container
+  prepare:
+     image: rancher/k3s-upgrade:latest
+     args: ["prepare","k3s-master-plan"]
+  nodeSelector:
+    matchExpressions:
+    - {key: k3s-worker-upgrade, operator: Exists}
+  serviceAccountName: system-upgrade
+  drain:
+    force: true
+  upgrade:
+    image: rancher/k3s-upgrade`
```

Comment:
If we don't use these plan constants can we just delete them?

Reply:
done